libs/blocks/chart/chart.js

```diff
@@ -203,10 +203,13 @@ const handleIntersect = (chartWrapper, chartType, data, colors, size) => (entrie
   });
 };
 
-const getColors = (authoredColor) => {
+export const getColors = (authoredColor) => {
   const colorList = Object.values(colorPalette);
 
-  if (isNullish(authoredColor) || !colorPalette.hasOwnProperty(authoredColor)) return colorList;
+  if (
+    isNullish(authoredColor)
+    || !Object.prototype.hasOwnProperty.call(colorPalette, authoredColor)
```

Comment:
It looks like `hasOwn` is recommended over `hasOwnProperty`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn
```suggestion
    || !Object.hasOwn(colorPalette, authoredColor)
```

Reply:
Great find, thank you.